doc/subsystems/bluetooth/gatt-pics.rst

```diff
@@ -20,8 +20,23 @@ Parameter Name	Selected	Description
 ==============	===========	============================================
 TSPC_GATT_1_1	True		Generic Attribute Profile Client (C.1)
 TSPC_GATT_1_2	True		Generic Attribute Profile Server (C.2)
-TSPC_GATT_1A_1	False (*)	Complete GATT client (C.3)
-TSPC_GATT_1A_2	False (*)	Complete GATT server (C.4)
+==============	===========	============================================
+
+
+GATT role configuration
+***********************
+
+==============	===========	============================================
+Parameter Name	Selected	Description
+==============	===========	============================================
+TSPC_GATT_1a_1	True		GATT Client over LE (C.1)
+TSPC_GATT_1a_2	False		GATT Client over BR/EDR (C.2)
+TSPC_GATT_1a_3	True		GATT Server over LE (C.3)
+TSPC_GATT_1a_4	False		GATT Server over BR/EDR (C.4)
+TSPC_GATT_1a_5	False		Complete GATT Client layer over LE (C.5)
+TSPC_GATT_1a_6	False		Complete GATT Client layer over BR/EDR (C.6)
+TSPC_GATT_1a_7	False		Complete GATT Server layer over LE (C.7)
```

Comment:
What exactly would it take to say that we have these "complete" layers?

Reply:
I have absolutely no idea :D I did not find anything about it neither in Core Spec nor Test Sepcification.  In ICS file is written that "Optional to support IF 1a/3 “GATT Server over LE” is supported, otherwise Excluded." only.
Toggling this in PTS doesn't enable/disable any of the test cases.
IMO this ICS does not makes sense.